Tasks/DownloadBuildArtifacts/task.json

```diff
@@ -148,12 +148,15 @@
     {
       "endpointId": "tfs:teamfoundation",
       "target": "definition",
-      "endpointUrl": "{{endpoint.url}}/{{project}}/_apis/build/definitions?$top=1000",
+      "endpointUrl": "{{endpoint.url}}/{{project}}/_apis/build/definitions?api-version=3.0-preview&$top=1000&continuationToken={{{continuationToken}}}&queryOrder=2",
```

Comment:
why are we adding api-version explicitly? I believe latest version of API is much higher